aya/src/programs/cgroup_skb.rs

```diff
@@ -155,6 +167,7 @@ impl Link for CgroupSkbLinkInner {
 }
 
 define_link_wrapper!(
+    /// An owned link, obtained using [CgroupSkb::forget_link]
```

Comment:
The description seems wrong? These are not owned, and can be returned by more than forget_link (eg see lirc). (Also missing period at the end of the sentence here and in other link docs)

I'd say:

```
The link used by [CgroupSkb] programs.
```

...or something